azure-pipelines.yml

```diff
@@ -57,6 +57,8 @@ jobs:
       set -e
       ruby -v
       echo -e "install: --no-document --env-shebang\nupdate:  --no-document --env-shebang" > ~/.gemrc
+      gem install bundler -v 2.3.18
+      bundle _2.3.18_ install
```

Comment:
why so? and why you haven't removed next line with simple `bundle install`?

Reply:
Because CI fails if the bundle version on agent is not matching the version in Gemfile. We either set non-strict version in Gemfile (~> 2.3), or use this construction to install the version we set in Gemfile.

As for the second `bundle install`, I think it should be removed. @IlyaBausovAkvelon.

Reply:
Removed `bundle install`